### PR TITLE
Chrome headless and binary opts can be passed in @sessions

### PR DESCRIPTION
**Pull request by dvic**
This makes it possible to override `headless` and `binary` for a particular test, e.g., 

```elixir
@sessions [[headless: false]]
feature "my feature", %{session: session} do 
  ...
end
```

### CODE REVIEW

**Comment by mhanberg** [comment]
Can you edit the Wallaby.Feature docs to mention you can pass these options?

**Comment by dvic** [comment]
> Can you edit the Wallaby.Feature docs to mention you can pass these options?

Done!

File: lib/wallaby/feature.ex [hunk 1]
```diff
@@ -69,17 +69,19 @@ defmodule Wallaby.Feature do
...
-    [capabilities: %{}]
+    [headless: false, binary: "some_path", capabilities: %{}]
   ]
   feature "test with different capabilities", %{session: session} do
     # ...
   end
   ```
 
+
+  You can also control
```
Comment (mhanberg) [inline lib/wallaby/feature.ex:84]: ?

Comment (dvic) [reply lib/wallaby/feature.ex]: Oops, sorry about that! (Initially I typed a separate section on it, but forgot to remove bits of that part 🤦 )

**Comment by mhanberg** [comment]
Have you tested this locally with a real project?

**Comment by dvic** [comment]
> Have you tested this locally with a real project?

Yup

**Comment by mhanberg** [comment]
Thanks!

**Pull request merged**